src/bytecode/instr.py

```diff
@@ -168,6 +168,24 @@ class Label:
     __slots__ = ()
 
 
+class TryBegin:
+    __slots__ = ("target", "push_lasti", "stack_depth")
+
+    def __init__(
+        self, target: Label, push_lasti: bool, stack_depth: int | _UNSET = UNSET
```

Comment:
I think the use of `|` instead of `Union` will make the code compatible with Python >= 3.10 (https://peps.python.org/pep-0604/). Is this intentional?

Reply:
No. I just prefer this syntax and forget the compatibility issue at the time of writing.